src/community/jdbcconfig/pom.xml

```diff
@@ -52,7 +52,7 @@
       <!-- blazing fast LZW compression/decompression -->
       <groupId>com.ning</groupId>
       <artifactId>compress-lzf</artifactId>
-      <version>0.9.1</version>
+      <version>1.0.3</version>
```

Comment:
Good point Jody. I was simply updating the version where it was defined, but I will move it to the root pom.